readthedocs/templates/builds/build_detail.html

```diff
@@ -76,7 +76,7 @@
               style="display: none;">
           {{ build.get_state_display }}
         </span>
-        <img src="{{ MEDIA_URL }}images/loader.gif"
```

Comment:
Agreed, I've been slowing moving things over, but an audit here would be great. I'll open an issue to track that work.